stable/phpmyadmin/templates/NOTES.txt

```diff
@@ -0,0 +1,36 @@
+1. Get the application URL by running these commands:
+{{- if .Values.ingress.enabled }}
+  You should be able to access your new PhpMyAdmin installation through
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ if $.Values.ingress.host }}{{.Values.ingress.host}}{{else}}your-cluster-ip{{end}}{{ $.Values.ingress.path }}
+  {{if not $.Values.ingress.host}}
+  Find out your cluster ip address by running:
+  $ kubectl cluster-info
+  {{end}}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "phpmyadmin.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "phpmyadmin.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "phpmyadmin.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "phpmyadmin.name" . }},release={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl port-forward $POD_NAME 8080:80
+{{- end }}
+
+2. How to log in
+{{ if or (.Values.db.suffix) (.Values.db.host)}}
+PhpMyAdmin has been configured to connect to a database in {{ if .Values.db.suffix}}{{template "phpmyadmin.dbfullname" .}}{{ else }}{{.Values.db.host}}{{end}}
+{{if .Values.db.port}}with port {{.Values.db.port}} {{end }}
+Please login using a database username and password. 
+{{ else }}
+PhpMyAdmin has not been configure to point to a specific database. Please provide the db host, username and password at log in or upgrade the release with 
+a specific database
+$ helm upgrade {{.Release.Name}} stable/phpmyadmin --set db.host=mydb 
+{{end}}
+
+** Please be patient while the chart is being deployed **
+
```

Comment:
file could use some formatting to make things more clear/readable

Reply:
done